sandcastle/api.py

```diff
@@ -255,14 +255,20 @@ def get_api_client() -> client.CoreV1Api:
         load service account identity, otherwise load kubeconfig
         """
         logger.debug("Initialize kubernetes client")
+        configuration = client.Configuration()
         if "KUBERNETES_SERVICE_HOST" in os.environ:
             logger.info("loading incluster config")
-            config.load_incluster_config()
+            config.load_incluster_config(client_configuration=configuration)
         else:
             logger.info("loading kubeconfig")
-            config.load_kube_config()
-        configuration = client.Configuration()
-        assert configuration.api_key
+            config.load_kube_config(client_configuration=configuration)
+        if not configuration.api_key:
+            raise SandcastleException(
+                f"We loaded the kube config but can't access any cluster. "
+                "We have this configutation:\n"
+                f"Auth: {configuration.auth_settings()}\n"
```

Comment:
What kind of information does this expose? Is there anything sensitive, like API keys or passwords?

Reply:
Good point.
Yes, it contains API key, but we raise this exception only `if not configuration.api_key` in which case the `auth_settings()` returns nothing valuable -> so we don't actually need it :-)
I'm removing it, thanks.

Reply:
I added this code because the assert config.api_key was not very useful while thing code giving actually some info that things are wrong

it's true though that content of those 2 method calls is empty/nonsense-ish, so +1 removing those